rpc/synchronization_forcer.go

```diff
@@ -0,0 +1,42 @@
+package rpc
+
+import (
+	"errors"
+
+	"github.com/TykTechnologies/tyk/apidef"
+	"github.com/TykTechnologies/tyk/storage"
+)
+
+type SyncronizerForcer struct {
+	store *storage.RedisCluster
+}
+
+func NewSyncForcer(redisController *storage.RedisController) *SyncronizerForcer {
+	sf := &SyncronizerForcer{}
+
+	sf.store = &storage.RedisCluster{KeyPrefix: "synchronizer-group-", RedisController: redisController}
+	sf.store.Connect()
+
+	return sf
+}
+
+func (sf *SyncronizerForcer) GrouLoginCallback(userKey string, groupID string) interface{} {
```

Comment:
fix typo from `GrouLoginCallback` to `GroupLoginCallback `

Reply:
whoop 🙈  fixed it